.circleci/config.yml

```diff
@@ -17,5 +17,6 @@ jobs:
           key: v1-dependencies-{{ checksum "package.json" }}-{{ checksum "yarn.lock" }}
       - run: yarn lint
       - run: yarn flow:ci
+      - run: yarn test --runInBand
```

Comment:
I decided to not run the coverage (yet?).

Also I'm not comfortable fetching a bash script from a web page to execute in an environment that will have secrets (in the future)...